R/random-forest-deployment.R

```diff
@@ -7,24 +7,43 @@
 #' \item Push these predictions to SQL Server
 #' }
 #' @docType class
-#' @usage RandomForestDeployment(type, df, grainCol, testWindowCol, 
-#' predictedCol, impute, debug)
+#' @usage RandomForestDeployment(type, df, grainCol, predictedCol, impute, debug, cores, modelName)
 #' @import caret
 #' @import doParallel
 #' @importFrom R6 R6Class
 #' @import ranger
 #' @param type The type of model (either 'regression' or 'classification')
 #' @param df Dataframe whose columns are used for calc.
 #' @param grainCol The dataframe's column that has IDs pertaining to the grain
-#' @param testWindowCol (depreciated) All data now receives a prediction
 #' @param predictedCol Column that you want to predict. If you're doing
 #' classification then this should be Y/N.
 #' @param impute For training df, set all-column imputation to F or T.
 #' This uses mean replacement for numeric columns
 #' and most frequent for factorized columns.
 #' F leads to removal of rows containing NULLs.
 #' @param debug Provides the user extended output to the console, in order
-#' to monitor the calculations throughout. Use T or F.
+#' @param cores Number of cores you'd like to use.  Defaults to 2.
```

Comment:
example indicates 1 core and it might be good for it to be consistent with default of 2.